diff --git a/board/xilinx/zynq/Makefile b/board/xilinx/zynq/Makefile
index f40fe382f57..9e267bae39e 100644
--- a/board/xilinx/zynq/Makefile
+++ b/board/xilinx/zynq/Makefile
@@ -9,7 +9,7 @@ ifneq ($(CONFIG_XILINX_PS_INIT_FILE),"")
 PS_INIT_FILE := $(shell cd $(srctree); readlink -f $(CONFIG_XILINX_PS_INIT_FILE))
 init-objs := ps_init_gpl.o
 spl/board/xilinx/zynq/ps_init_gpl.o board/xilinx/zynq/ps_init_gpl.o: $(PS_INIT_FILE)
-	$(CC) $(c_flags) -I $(srctree)/$(src) -c -o $@ $^
+	$(CC) $(c_flags) -I $(srctree)/$(src) -Wno-strict-prototypes -c -o $@ $^
 endif
 
 DEVICE_TREE ?= $(CONFIG_DEFAULT_DEVICE_TREE:"%"=%)
